neard/src/cli.rs

```diff
@@ -146,7 +149,8 @@ struct NeardOpts {
     /// Directory for config and data.
     #[clap(long, parse(from_os_str), default_value_os = crate::DEFAULT_HOME.as_os_str())]
     home: PathBuf,
-    /// Skips consistency checks of the 'genesis.json' file upon startup.
+    /// Skips consistency checks of the config files including
+    /// genesis.json, config.json, node_key.json and validator_key.json upon startup.
```

Comment:
Looking at `fn load_config()` I see that `config.json` is always validated, isn't it?

Reply:
forgot to change comments here. Done.